events/2025-11-24-gat-brno.md

```diff
@@ -0,0 +1,353 @@
+
+Galaxy is an open web based platform for data integration and analysis. It is deployed at large and small organizations around the world and used in a broad range of research domains.
+
+## Schedule
+
+Please see the Program tab
+
+### Prerequisites
+
+**Please review the prerequisites carefully before registering**. If you arrive without meeting the prerequisites then the workshop will be a frustrating experience.
+
+#### Comfortable with the Unix/Linux command line
+
+Most of the workshop will happen at the Linux command line. If you aren’t comfortable with this before you arrive, then you will be lost the entire time. How comfortable do you need to be? Here’s a sampling of commands and concepts that you should be comfortable with:
+
+```
+cd, mkdir, rmdir, ls
+tar, gzip
+cp, mv, rm, chmod, ln
+~, .., ./, /tmp,
+cat, tail, less
+man
+```
+
+Note: *This is not an exhaustive list (or even a proper subset) of the commands that will be used in the workshop. They are included here to give you an idea of the depth of knowledge that is required.*
+
+We have a few tutorials that can help you get refreshed if you need:
+
+- [CLI Basics]({% link topics/data-science/tutorials/cli-basics/tutorial.md %})
+- [CLI Advanced]({% link topics/data-science/tutorials/cli-advanced/tutorial.md %})
+
+#### Competent in a Unix/Linux text editor
+
+Linux has several text editors available in it. You'll need to be comfortable in at least one of them before the workshop starts. Organizers will make sure that these editors are available on the Linux images we use in the workshop:
+
+We recommend that you use [Nano](https://www.nano-editor.org/), if you don't know another text editor. It's [simple and easy to use](https://www.howtogeek.com/42980/the-beginners-guide-to-nano-the-linux-command-line-text-editor/). If you know another editor like [Emacs](https://www.gnu.org/software/emacs/) or [VIM](http://www.vim.org/) you are of course welcome to use those as well.
+
+#### Some experience with Linux Package Management
+
+You don't need to be an expert at Linux package management, but you do need to have some experience at installing and upgrading packages on a Linux system.
+
+We'll be using Ubuntu based images during the workshop, which means we’ll be using [**apt**](https://help.ubuntu.com/community/AptGet/Howto) to do package management. Get to know it.
+
+Prerequisites include all of the Introduction Session prerequisites, plus familiarity with the topics presented in that session.
```

Comment:
most of this isn't necessary since it'll be in the relevant tabs.

Reply:
@hexylena thank you, I did that, hopefully more compact now without duplications